UnityProject/Assets/Scripts/Objects/IBumpObject.cs

```diff
@@ -0,0 +1,9 @@
+﻿using UnityEngine;
+
+namespace Objects
+{
+	public interface IBumpObject
```

Comment:
I don't know why I didn't mention it before, but the interface should be called ``IBumpableObject``to be ultra complacent with naming convention.

> Another common interface naming convention is used when an interface declares only one key method, such as Draw() in the above example. The interface name is then formed as an adjective by adding the "...able" suffix. So, the interface name above could also be IDrawable. This convention is used throughout the .NET Framework.

https://en.wikibooks.org/wiki/C_Sharp_Programming/Interfaces

Reply:
cool, ive renamed it